wallet/tests/restore.rs

```diff
@@ -366,10 +366,10 @@ fn perform_restore(test_dir: &str) -> Result<(), libwallet::Error> {
 fn wallet_restore() {
 	let test_dir = "test_output/wallet_restore";
 	if let Err(e) = setup_restore(test_dir) {
```

Comment:
Why not `setup_restore(test_dir).expect("Libwallet Error")`?  It is shorter and does almost the same, panic will print out the stacktrace. Also below